using-mobile-extensions/adobe-experience-platform-messaging/README.md

```diff
@@ -0,0 +1,211 @@
+
+1. In Experience Platform Launch, in your mobile property, click the **Extensions** tab.
+2. On the **Catalog** tab, locate or search for the **Adobe Experience Platform - Messaging** extension, and click **Install**.
+3. Select the **Event Dataset** for Production, Stage and Development Environments. The datasets selected should use a schema that uses the Push Notification Tracking mixin. For more information, see [Setup Schemas & Datasets](../../getting-started/configure-schema-and-dataset.md).
+4. Click **Save**.
+5. Follow the publishing process to update SDK configuration.
+![AEP Messaging extension configuration](../../.gitbook/assets/mobile-edge-consent-launch-configuration.png)
+
+## Configure Application Configuration
+1. In Experience Platform Launch, click the **App Configurations** tab on the left pane.
+2. Click the **Add App Configuration** button.
+3. Provide a **Name** and select the **Messaging Service Type**
+![AEP App configuration](../../.gitbook/assets/add-app-config.png)
+
+{% tabs %}
+{% tab title="Android" %}
+
+4. Provide the **App ID (Android package name)**. Usually the package name is the app id in your `build.gradle` file.
+5. Drag and drop the FCM push credentials. For more details on how to get the push credentials follow [this](https://firebase.google.com/docs/admin/setup#initialize-sdk).
+ 
+![AEP Android App Configuration](../../.gitbook/assets/add-app-config-android.png)
+
+{% endtab %}
+
+{% tab title="iOS" %}
+
+4. Provide the **App ID (iOS Bundle ID)**. Bundle Id can be found in the `General` tab of the primary target in **XCode**.
+5. Drag and drop the **Apple Push Notification Authentication Key** for your Apple developer account. This key can be acquired from the **Certificates, Identifiers & Profiles** page. 
+6. Provide the **Key ID**. This is a 10 character string assigned during the creation of p8 auth key. It can be found under **Keys** tab in **Certificates, Identifiers & Profiles** page.
+7. Provide the **Team ID**. This is a string value which can be found under the Membership tab.
+
+![AEP Android App Configuration](../../.gitbook/assets/add-app-config-ios.png)
+
+For more information follow this [link](https://help.apple.com/developer-account/#/deva05921840)
+
+{% endtab %}
+{% endtabs %}
+
+## Configure the application to allow push notification 
+{% tabs %}
+{% tab title="Android" %}
+To add Firebase Messaging Service to your app please follow the [link](https://firebase.google.com/docs/cloud-messaging/android/client)
+
+{% endtab %}
+
+{% tab title="iOS" %}
+To add APNs to your app please follow the [link](https://developer.apple.com/documentation/usernotifications/registering_your_app_with_apns)
+
+{% endtab %}
+{% endtabs %}
+
+## Add the AEP Messaging extension to your application
+
+### Download and import the Messaging extension
+
+{% tabs %}
+{% tab title="Android" %}
+
+### Java
+
+1. Add the Mobile Core, Edge and Messaging extensions to your project using the app's Gradle file.
```

Comment:
should you add `EdgeIdentity` to this list, since it's represented in the sample code?